index.html

```diff
@@ -10,15 +10,51 @@
     crossorigin="anonymous"
     href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
     integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==">
+  <link href="https://fonts.googleapis.com/css2?family=Proza+Libre&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
-  </header>
-  <main>
-  </main>
+  <div class="center">
+
+    <header class="headers">
+      <nav class="container row headers full-width header-position">
+        <h1 class="headers main-header headings">Code Journal</h1>
+      </nav>
+    </header>
+
+    <main class="container full-width">
```

Comment:
What is this `full-width` class you are using? I don't see it in your styling anywhere. Do you mean to be using the `column-full` class we prepared for you in `layout.css`?